src/offset/local/unix.rs

```diff
@@ -89,14 +89,20 @@ fn fallback_timezone() -> Option<TimeZone> {
 impl Default for Cache {
     fn default() -> Cache {
         // default to UTC if no local timezone can be found
+        let env_tz = env::var("TZ").ok();
+        let env_ref = env_tz.as_ref().map(|s| s.as_str());
         Cache {
-            zone: TimeZone::local().ok().or_else(fallback_timezone).unwrap_or_else(TimeZone::utc),
-            source: Source::default(),
             last_checked: SystemTime::now(),
+            source: Source::new(env_ref),
+            zone: new_zone(env_ref),
         }
     }
 }
 
+fn new_zone(var: Option<&str>) -> TimeZone {
```

Comment:
I find the `new_` prefix a little confusing here. In the end this is just the current zone, right? Maybe just call this `current()` or `zone()` or `current_zone()`?

Reply:
good call, I've renamed to `current_zone`